senz-web/backend/routes/userRoutes.js

```diff
@@ -6,6 +6,7 @@ const jwt = require("jsonwebtoken");
 const config = require("config");
 const jwtVerify = require("./verifyTokens");
 const uuidv4 = require("uuid/v4");
+var validator = require("email-validator");
```

Comment:
const should be used instead of var

Reply:
Okay , I have made the required changes.